EarlGrey/Common/GREYAnalytics.m

```diff
@@ -128,4 +144,26 @@ + (void)trackEventWithTrackingID:(NSString *)trackingID
   }] resume];
 }
 
+#pragma mark - Private
+
+/**
+ *  Usage data is sent via Google Analytics indicating completion of a test case, if a delegate is
+ *  specified it is invoked to handle the analytics instead.
+ */
+- (void)grey_testCaseInstanceDidTearDown {
+  if (trackAnalyticsOnNextTearDown) {
+    trackAnalyticsOnNextTearDown = NO;
```

Comment:
Please add a comment that says: Reset var to track multiple test case invocations.